frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/FeatureOverviewEnvironment.tsx

```diff
@@ -134,13 +134,7 @@ const FeatureOverviewEnvironment = ({
 
     return (
         <ConditionallyRender
-            condition={
-                !new Set(
-                    Array.isArray(globalStore.hiddenEnvironments)
-                        ? globalStore.hiddenEnvironments
-                        : [],
-                ).has(env.name)
-            }
+            condition={!new Set(globalStore.hiddenEnvironments).has(env.name)}
```

Comment:
Is there any specific reason why we're initializing a new set here instead of doing `globalStore.hiddenEnvironments.includes(env.name)`?

Reply:
Reverted old code from before Friday. But I think your suggestion would work fine